Templates/Applications/Office365.json

```diff
@@ -471,7 +488,16 @@
     {
       "controlType": "input",
       "inline": true,
-      "noFieldset": true
+      "noFieldset": true,
+      "validate:i": 
```

Comment:
I don't think you need the interpolation flag here, right?

Reply:
Ah you are right, I copied this with the i :P